packages/flx/lib/signing.dart

```diff
@@ -165,8 +165,8 @@ ECSignature _asn1ParseSignature(Uint8List signature) {
 }
 
 PublicKey _publicKeyFromPrivateKey(ECPrivateKey privateKey) {
-  ECPoint Q = privateKey.parameters.G * privateKey.d;
-  return new ECPublicKey(Q, privateKey.parameters);
+  ECPoint q = privateKey.parameters.G * privateKey.d;
```

Comment:
Yeah.  That fix just hasn't made it's way into the SDK yet.  Should be in the next roll though.